R/environment.R

```diff
@@ -87,28 +87,16 @@ r_environment <- function(name, version = NULL,
   }
 
   if (is.null(custom_docker_image)) {
-    processor <- "cpu"
     if (use_gpu) {
-      processor <- "gpu"
+      base_image_with_address <- paste0("mcr.microsoft.com/azureml/base-",
```

Comment:
Looks like `base_image_with_address `is not set when `custom_docker_image `is not null?

Reply:
added